core/controllers/android_e2e_config.py

```diff
@@ -161,6 +161,19 @@ def post(self):
             '%s%d' % (story_domain.NODE_ID_PREFIX, 1),
             exp_id
         )
+
+        # Save the dummy image to the filesystem to be used as thumbnail.
```

Comment:
@anandwana001 do you need this twice? It's already in the file. (See a similar comment by @BenHenning in Srikanth's PR.)

Reply:
done